### PR TITLE
Activity chat

### PR DESCRIPTION
**Pull request by JadaTBK**
I'm Grant

^ this was written by jake

to test:
- join an activity
- toggle the mini / full description
- send a message with one account
- send a message with a different account in the same activity

### CODE REVIEW

**Review by JakeThurman — APPROVED** [review]
LGTM 

but: DO NOT MERGE until Noah's PR is merged.

**Comment by noahtallen** [comment]
@gyurisic I just rebased this on top of master for you

**Pull request merged**
